common/src/importers/onepasswordImporters/onepassword1PuxImporter.ts

```diff
@@ -0,0 +1,636 @@
+      cipher.identity.country = fieldValue;
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.state) && field.id === "state") {
+      cipher.identity.state = fieldValue;
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.licenseNumber) && field.id === "number") {
+      cipher.identity.licenseNumber = fieldValue;
+      return true;
+    }
+
+    return false;
+  }
+
+  private fillOutdoorLicense(field: FieldsEntity, fieldValue: string, cipher: CipherView): boolean {
+    if (this.isNullOrWhitespace(cipher.identity.firstName) && field.id === "name") {
+      this.processFullName(cipher, fieldValue);
+      return true;
+    }
+
+    // TODO ISO code
+    if (this.isNullOrWhitespace(cipher.identity.country) && field.id === "country") {
+      cipher.identity.country = fieldValue;
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.state) && field.id === "state") {
+      cipher.identity.state = fieldValue;
+      return true;
+    }
+
+    return false;
+  }
+
+  private fillMembership(field: FieldsEntity, fieldValue: string, cipher: CipherView): boolean {
+    if (this.isNullOrWhitespace(cipher.identity.firstName) && field.id === "member_name") {
+      this.processFullName(cipher, fieldValue);
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.company) && field.id === "org_name") {
+      cipher.identity.company = fieldValue;
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.phone) && field.id === "phone") {
+      cipher.identity.phone = fieldValue;
+      return true;
+    }
+
+    return false;
+  }
+
+  private fillPassport(field: FieldsEntity, fieldValue: string, cipher: CipherView): boolean {
+    if (this.isNullOrWhitespace(cipher.identity.firstName) && field.id === "fullname") {
+      this.processFullName(cipher, fieldValue);
+      return true;
+    }
+
+    // TODO Iso
+    if (this.isNullOrWhitespace(cipher.identity.country) && field.id === "issuing_country") {
+      cipher.identity.country = fieldValue;
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.passportNumber) && field.id === "number") {
+      cipher.identity.passportNumber = fieldValue;
+      return true;
+    }
+
+    return false;
+  }
+
+  private fillRewardsProgram(field: FieldsEntity, fieldValue: string, cipher: CipherView): boolean {
+    if (this.isNullOrWhitespace(cipher.identity.firstName) && field.id === "member_name") {
+      this.processFullName(cipher, fieldValue);
+      return true;
+    }
+
+    if (this.isNullOrWhitespace(cipher.identity.company) && field.id === "company_name") {
+      cipher.identity.company = fieldValue;
+      return true;
+    }
+
+    // if (this.isNullOrWhitespace(cipher.identity.licenseNumber) && field.id === "membership_no") {
+    //   cipher.identity.licenseNumber = fieldValue;
+    //   return true;
+    // }
```

Comment:
Should this be uncommented or can we remove it?

Reply:
I had the idea to map the membership-no into the licenseNumber field, but decided to keep it in the custom-fields along with the other unmapped ones.